plugins/apps/post-delete

```diff
@@ -1,4 +1,12 @@
 #!/usr/bin/env bash
 set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
+source "$PLUGIN_PATH/common/functions"
+APP="$1"; IMAGE_REPO=$(get_app_image_repo $APP)
 
-[[ -n $1 ]] && rm -r "$DOKKU_ROOT/$1" > /dev/null
+[[ -n $APP ]] && rm -r "$DOKKU_ROOT/$APP" > /dev/null
+
+# remove all application containers & images
+# shellcheck disable=SC2046
```

Comment:
without shellcheck spits out:
```
In ./plugins/apps/post-delete line 9:
docker rm -f $(docker ps -a --no-trunc | egrep "dokku/${APP}:" | awk '{ print $1 }' | xargs) &>/dev/null || true
             ^-- SC2046: Quote this to prevent word splitting.


In ./plugins/apps/post-delete line 10:
docker rmi $(docker images -q $IMAGE_REPO | xargs) &>/dev/null || true
           ^-- SC2046: Quote this to prevent word splitting.
```

I remember there being a bug at some point if we quoted it.